src/components/AccountDrawer/MiniPortfolio/Activity/parseRemote.tsx

```diff
@@ -165,6 +165,7 @@ function parseSwap(changes: TransactionChanges) {
       return {
         title: getSwapTitle(sent, received),
         descriptor: getSwapDescriptor({ tokenIn: sent.asset, inputAmount, tokenOut: received.asset, outputAmount }),
+        currencies: [gqlToCurrency(sent.asset), gqlToCurrency(received.asset)],
```

Comment:
I'm adding `currencies` to all of the activities objects so that `PortfolioLogo` can render the symbol in a missing token logo, if no logo is provided by the BE or FE